doc/sphinx/install.rst

```diff
@@ -60,6 +60,7 @@ to the channels, and then we can immediately install pagmo:
 .. code-block:: console
 
    $ conda config --add channels conda-forge
+   $ conda config --set channel_priority strict
```

Comment:
out of curiosity why was this needed?

Reply:
This was added recently to the conda-forge instructions on their website.